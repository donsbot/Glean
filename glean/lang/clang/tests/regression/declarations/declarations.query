query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
