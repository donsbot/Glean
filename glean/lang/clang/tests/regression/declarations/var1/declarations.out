[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 1,
      "lineEnd": 9,
      "columnEnd": 13
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 1,
      "lineEnd": 10,
      "columnEnd": 6
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 1,
      "lineEnd": 11,
      "columnEnd": 13
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 12,
      "columnBegin": 1,
      "lineEnd": 12,
      "columnEnd": 13
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 13,
      "columnBegin": 1,
      "lineEnd": 13,
      "columnEnd": 20
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 15,
      "columnBegin": 1,
      "lineEnd": 20,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 16,
      "columnBegin": 3,
      "lineEnd": 16,
      "columnEnd": 15
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 17,
      "columnBegin": 3,
      "lineEnd": 17,
      "columnEnd": 15
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 18,
      "columnBegin": 3,
      "lineEnd": 18,
      "columnEnd": 22
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 19,
      "columnBegin": 3,
      "lineEnd": 19,
      "columnEnd": 29
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 22,
      "columnBegin": 1,
      "lineEnd": 22,
      "columnEnd": 9
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 23,
      "columnBegin": 1,
      "lineEnd": 23,
      "columnEnd": 16
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 25,
      "columnBegin": 1,
      "lineEnd": 29,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 25,
      "columnBegin": 8,
      "lineEnd": 25,
      "columnEnd": 13
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 26,
      "columnBegin": 3,
      "lineEnd": 26,
      "columnEnd": 15
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 27,
      "columnBegin": 3,
      "lineEnd": 27,
      "columnEnd": 8
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 28,
      "columnBegin": 3,
      "lineEnd": 28,
      "columnEnd": 3
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 28,
      "columnBegin": 4,
      "lineEnd": 28,
      "columnEnd": 7
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 28,
      "columnBegin": 8,
      "lineEnd": 28,
      "columnEnd": 11
    }
  }
]